### DISCUSSION TITLE
po-multiselect: [p-filter-service] não funciona mais?

### DISCUSSION THREAD

**Discussion by leticiatomasio** [po-ui/po-angular#982] [Tire suas dúvidas] [answered]
A funcionalidade de serviço foi retirada do PoMultiselect? Tenho tentado implementar como a documentação orienta e acabo dando de cara com esses problemas:

![Captura de tela de 2021-08-23 14-14-15](https://user-images.githubusercontent.com/69875352/130490622-b58019f2-94ab-439c-9545-e691b7c48499.png)

![Captura de tela de 2021-08-23 14-26-55](https://user-images.githubusercontent.com/69875352/130490813-dbdd938d-6cb2-4a9c-9da3-3f4132e3b345.png)

**Comment by jhonyeduardo** [comment]
Boa tarde @leticiatomasio 

Qual versão instalada do po-ui/ng-componentes no seu package.json?

**Reply by leticiatomasio** [reply]
Boa tarde!

`"@po-ui/ng-components": "5.8.0"`

**Comment by felipepetuco** [comment, marked as answer]
Olá @leticiatomasio atualmente estamos na versão 15.13.0, e o recurso continua disponível.
Estou encerrando esta dúvida dado ao tempo que ela já permaneceu aberta.
Ficamos a disposição :)
